extensions/google-chrome/package.json

```
@@ -0,0 +1,45 @@
+{
+  "name": "google-chrome-go-to-tab",
+  "title": "Google Chrome Go to Tab",
```

Would `Google Chrome Tab Search` be a better name?

```suggestion
  "title": "Google Chrome Tab Search",
```